theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -755,7 +759,8 @@ def test_get_rid_of_advanced_indexing_version_of_xent(self):
         for expr in expressions:
             # Verify the optimizer worked on the expressions
             f = theano.function([x, y], expr, mode=mode)
-            self.assertTrue(hasattr(f.maker.fgraph.outputs[0].tag, 'trace'))
+            assert check_stack_trace(
+                f, ops_to_check=crossentropy_softmax_argmax_1hot_with_bias)
```

Comment:
If all the node in the original graph pattern have the same stack trace (on the same line) then I think it make sense to put that stack trace to all new nodes in the new pattern.

Not sure how hard it is to do or if this is worth it.